server/backend/database/testcases/testcases.go

```diff
@@ -1074,3 +1026,86 @@ func RunIsDocumentAttachedTest(t *testing.T, db database.Database, projectID typ
 		assert.False(t, attached)
 	})
 }
+
+// RunFindNextNCyclingProjectInfosTest runs the FindNextNCyclingProjectInfos tests for the given db.
+func RunFindNextNCyclingProjectInfosTest(t *testing.T, db database.Database) {
+	t.Run("`RunFindNextNCyclingProjectInfos` should search projects cyclic", func(t *testing.T) {
```

Comment:
Consider matching test name convention with other test names for now.
ex) "FindDeactivateCandidatesPerProject candidate search test"